src/MCPServer/lib/jobChainLink.py

```diff
@@ -35,19 +35,23 @@
 
 from databaseFunctions import auto_close_db, logJobCreatedSQL, getUTCDate
 
-from main.models import Job, MicroServiceChainLink, MicroServiceChainLinkExitCode, TaskType
+from main.models import Job, MicroServiceChainLink, MicroServiceChainLinkExitCode
+
 
 LOGGER = logging.getLogger('archivematica.mcp.server')
 
-# Constants
-constOneTask = TaskType.objects.get(description="one instance").pk
-constTaskForEachFile = TaskType.objects.get(description="for each file").pk
-constSelectPathTask = TaskType.objects.get(description="get user choice to proceed with").pk
-constGetReplacementDic = TaskType.objects.get(description="get replacement dic from user choice").pk
-constlinkTaskManagerGetMicroserviceGeneratedListInStdOut = TaskType.objects.get(description="Get microservice generated list in stdOut").pk
-constlinkTaskManagerGetUserChoiceFromMicroserviceGeneratedList = TaskType.objects.get(description="Get user choice from microservice generated list").pk
-constlinkTaskManagerSetUnitVariable = TaskType.objects.get(description="linkTaskManagerSetUnitVariable").pk
-constlinkTaskManagerUnitVariableLinkPull = TaskType.objects.get(description="linkTaskManagerUnitVariableLinkPull").pk
```

Comment:
I had the following scenario:
1. Wrote a `test_package.py` with a simple unit test as a pytest function.
2. `test_package.py` imports `package.py` which would import this module as a side effect.
3. `jobChainLink.py` (this snippet above) this the database which pytest has not loaded yet.